### ISSUE TITLE
registerDoRNG() causes "object not found" error

### ISSUE THREAD

**Issue by hanase** [renozao/doRNG#7]
Hello @renozao ,
I found that after calling registerDoRNG(), objects passed to the foreach-%dopar% loop are not found. Here is an example using doFuture (it behaves the same with doParallel):

```
library(doFuture)
library(doRNG)

doFut <- function(nodes, n, mu = 1, seed = 1) {
    cl <- makeCluster(nodes)
    registerDoFuture()
    plan(cluster, workers = cl)
    registerDoRNG(seed)
    x <- foreach(i = 1:n) %dopar% rnorm(i, mean = mu)
    stopCluster(cl)
    x
}
doFut(2, 10)
```
This fails with
`Error in { : task 1 failed - "object 'mu' not found"`

If the `registerDoRNG` line is commented out, it works. It also works with the `%dorng%` operator. Is there anything in the code I'm missing that could make it work?

Here is my sessionInfo():

```
R version 3.4.0 (2017-04-21)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.6

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_IE.UTF-8/en_IE.UTF-8/en_IE.UTF-8/C/en_IE.UTF-8/en_IE.UTF-8

attached base packages:
[1] parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] doRNG_1.6.6     rngtools_1.2.4  pkgmaker_0.22   registry_0.5    doFuture_0.6.0 
[6] iterators_1.0.9 foreach_1.4.4   future_1.7.0   

loaded via a namespace (and not attached):
 [1] codetools_0.2-15 listenv_0.7.0    digest_0.6.15    xtable_1.8-2     magrittr_1.5    
 [6] stringi_1.1.7    tools_3.4.0      stringr_1.3.0    compiler_3.4.0   globals_0.11.0  
```

**Comment by renozao** [comment]
This should be fixed by 5baa6b4.
Can you please try using the github version:
```R
devtools::install_github("renozao/doRNG")
```

**Comment by hanase** [comment]
Yes - that fixed it. Thanks!
